mmv1/products/privateca/terraform.yaml

```diff
@@ -15,51 +15,28 @@
 overrides: !ruby/object:Overrides::ResourceOverrides
   CertificateAuthority: !ruby/object:Overrides::Terraform::ResourceOverride
     autogen_async: true
-    iam_policy: !ruby/object:Api::Resource::IamPolicy
-      parent_resource_attribute: certificate_authority
-      allowed_iam_role: 'roles/privateca.certificateManager'
-      method_name_separator: ':'
-      example_config_body: 'templates/terraform/iam/example_config_body/privateca_certificate_authority.tf.erb'
-      base_url: '{{name}}'
-    description: |
-      {{description}}
-      
-      ~> **Warning:** Please remember that all resources created during preview (via the terraform-provider-google-beta)
-      will be deleted when CA service transitions to General Availability (GA). Relying on these
-      certificate authorities for production traffic is discouraged.
-    import_format: ["projects/{{project}}/locations/{{location}}/certificateAuthorities/{{certificate_authority_id}}"]
+    import_format: ["projects/{{project}}/locations/{{location}}/caPools/{{pool}}/certificateAuthorities/{{certificate_authority_id}}"]
```

Comment:
Oh weird- is this not implied by the `self_link`?

Reply:
It defaults to {{name}} at the end rather than certificate_authority_id. Not sure why we chose to go with that in the first place, this could be a chance to change it as we are breaking this resource anyways

Reply:
Oh! I thought the default import format used the self link, cool.